src/main/java/com/disi/TravelPoints/model/User.java

```diff
@@ -20,57 +28,6 @@ public class User {
     @JsonIgnore
```

Comment:
Do we need this annotation?

Reply:
Yes, when you do a get for user to avoid an infinite loop